### PR TITLE
Add compat data for Worker (Worker API)

### PR DESCRIPTION
**Pull request by maboa**
https://developer.mozilla.org/en-US/docs/Web/API/Worker

### CODE REVIEW

File: api/Worker.json [hunk 1]
```diff
@@ -0,0 +1,337 @@
+{
+  "api": {
+    "Worker": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/Worker",
+        "support": {
+          "webview_android": {
+            "version_added": "4.4"
```
Comment (jpmedley) [inline api/Worker.json]: Change this to `true`. Webview version numbers follow Chrome, not Android.

File: api/Worker.json [hunk 2]
```diff
@@ -0,0 +1,337 @@
...
+          },
+          "chrome": {
+            "version_added": "4"
+          },
+          "chrome_android": {
+            "version_added": null
```
Comment (jpmedley) [inline api/Worker.json]: Change to `true`.

File: api/Worker.json [hunk 3]
```diff
@@ -0,0 +1,337 @@
...
+          "deprecated": false
+        }
+      },
+      "Worker": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Worker/Worker",
+          "description": "<code>Worker()</code> constructor",
+          "support": {
+            "webview_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/Worker.json]: `true`

File: api/Worker.json [hunk 4]
```diff
@@ -0,0 +1,337 @@
...
+            },
+            "chrome": {
+              "version_added": "4"
+            },
+            "chrome_android": {
+              "version_added": "4.4"
```
Comment (jpmedley) [inline api/Worker.json]: `true`

File: api/Worker.json [hunk 5]
```diff
@@ -0,0 +1,337 @@
...
+          "description": "Constructor <code>name</code> option",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
```
Comment (jpmedley) [inline api/Worker.json]: Should be `false` for all 3 versions of Chrome.

File: api/Worker.json [hunk 6]
```diff
@@ -0,0 +1,337 @@
...
+              "version_added": "55"
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": null
+            },
+            "opera_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/Worker.json]: Should be `false` for both versions of Opera.

File: api/Worker.json [hunk 7]
```diff
@@ -0,0 +1,337 @@
...
+            }
+          }
+        }
+      },
+      "onmessage": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Worker/onmessage",
+          "support": {
+            "webview_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/Worker.json]: `true`

File: api/Worker.json [hunk 8]
```diff
@@ -0,0 +1,337 @@
...
+            },
+            "chrome": {
+              "version_added": "4"
+            },
+            "chrome_android": {
+              "version_added": "4.4"
```
Comment (jpmedley) [inline api/Worker.json]: `true`

File: api/Worker.json [hunk 9]
```diff
@@ -0,0 +1,337 @@
...
+            "deprecated": false
+          }
+        }
+      },
+      "onmessageerror": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Worker/onmessageerror",
+          "support": {
+            "webview_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/Worker.json]: 60

File: api/Worker.json [hunk 10]
```diff
@@ -0,0 +1,337 @@
...
+            "deprecated": false
+          }
+        }
+      },
+      "terminate": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Worker/terminate",
+          "support": {
+            "webview_android": {
+              "version_added": "4.4"
```
Comment (jpmedley) [inline api/Worker.json]: `true`

File: api/Worker.json [hunk 11]
```diff
@@ -0,0 +1,337 @@
...
+            },
+            "chrome": {
+              "version_added": "4"
+            },
+            "chrome_android": {
+              "version_added": "4.4"
```
Comment (jpmedley) [inline api/Worker.json]: `true`

**Comment by maboa** [comment]
That should be those changes made now. Thanks :)

File: api/Worker.json [hunk 12]
```diff
@@ -0,0 +1,337 @@
...
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "name": {
```
Comment (Elchi3) [inline api/Worker.json]: `name` needs to be nested under the `Worker` constructor

**Comment by maboa** [comment]
Hopefully fixed now :)

**Pull request merged**